app/page.tsx

```diff
@@ -49,19 +49,19 @@ export default function Home() {
 
   return (
     <div className="relative h-full flex flex-col justify-center items-center">
-      <div className="absolute items-center space-x-1 top-4 left-4 text-gray-200 text-lg font-semibold">
+      <div className="absolute items-center space-x-1 top-4 left-4 text-lg font-semibold">
         <DropdownMenu>
           <DropdownMenuTrigger asChild>
-            <button className="flex items-center px-4 py-2 rounded-md text-lg focus:outline-none hover:bg-zinc-900">
+            <div className="px-4 py-2 rounded-md cursor-pointer text-center hover:bg-gray-200 dark:hover:bg-gray-700">
```

Comment:
Do not use tailwindcss' `dark` prefix. Use colors from`app/globals.css` for different color mode.